docs/server/source/production-deployment-template/node-on-kubernetes.rst

```diff
@@ -128,7 +128,16 @@ Step 4.1: Vanilla NGINX
      the ConfigMap followed by ``-dep``. For example, if the value set in the
      ``ngx-instance-name`` is ``ngx-instance-0``, set  the
      ``spec.selector.app`` to ``ngx-instance-0-dep``.
-   
+     
+   * Set ``ngx-public-mdb-port`` to 27017, or the port number on which you
+     want to expose MongoDB service.
```

Comment:
Even I couldn't find a better way to write that. And that made sense intuitively.